File: Programming Basics with Python/02 Conditional statements/More exercises/02_Sleepy_Tom_Cat.py
holidays = int(input())
work_days_play = (365 - holidays) * 63
holidays_play = holidays * 127
play_time_in_minutes = work_days_play + holidays_play
result = 30000 - play_time_in_minutes
if play_time_in_minutes >= 30000:
    print('Tom will run away')
    print(
        f'{abs(result) // 60} hours and {abs(result) % 60} minutes more for play')
else:
    print('Tom sleeps well')
    print(f'{result // 60} hours and {result % 60} minutes less for play')

File: JS Basics/02 Conditional Statements - More Exercises/02. Sleepy Tom Cat.js
function tom(days) {
    let workDays = 365 - Number(days);
    let playMinutes = workDays * 63 + Number(days) * 127;
    let result = Math.abs(30000 - playMinutes);
    if (30000 > playMinutes) {
        console.log("Tom sleeps well");
        console.log(`${Math.floor(result / 60)} hours and ${result % 60} minutes less for play`);
    }
    else{
        console.log("Tom will run away");
        console.log(`${Math.floor(result / 60)} hours and ${result % 60} minutes more for play`);
    }
}
tom(20)